### PR TITLE
Add @ApiOperation vs @ApiResponse in Swagger

### PR DESCRIPTION
**Pull request by apeterlic**

### CODE REVIEW

File: spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/response/CustomerResponse.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package com.baeldung.swagger_responses.response;
+
+public class CustomerResponse {
```
Comment (andrzejsydor) [inline spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/response/CustomerResponse.java:3]: Couldn't be immutable class?

File: spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/controller/CustomerController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+public class CustomerController {
+
+    private final CustomerService customerService;
+
+    public CustomerController(CustomerService customerService) {
+        this.customerService = customerService;
+    }
+
+    @ApiOperation(value = "Gets customer by ID", response = CustomerResponse.class, notes = "Customer must exist")
+    @ApiResponses(value = { @ApiResponse(code = 400, message = "Invalid ID supplied"), @ApiResponse(code = 404, message = "Customer not found"), @ApiResponse(code = 500, message = "Internal server error", response = ErrorResponse.class) })
```
Comment (andrzejsydor) [inline spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/controller/CustomerController.java:30]: The `ApiResponse`s should be in new for better readability.

Comment (apeterlic) [reply spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/controller/CustomerController.java:30]: In the new line?

File: spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/response/CustomerResponse.java (hunk 1 above)
Comment (apeterlic) [reply spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/response/CustomerResponse.java:3]: Sure, I will make the class immutable.

File: spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/SwaggerDemoApplication.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.baeldung.swagger_responses;
```
Comment (JavierPeris) [inline spring-boot-modules/spring-boot-swagger/src/main/java/com/baeldung/swagger_responses/SwaggerDemoApplication.java:1]: @apeterlic although Java naming convention allows underscore for package naming we avoid them, so please rename it, you can just use swaggerresponses.

**Pull request merged**